src/.vuepress/config.js

```diff
@@ -35,7 +35,6 @@ module.exports = {
           { text: 'Get Started', link: '/docs/get-started/' },
           { text: 'Use cases', link: '/docs/use-cases/' },
           { text: 'Concepts', link: '/docs/concepts/' },
-          { text: 'Tutorials', link: '/docs/tutorials/' },
```

Comment:
did we replace tutorials by something else?

Reply:
It was moved into the getting started section awhile ago. This was just not cleaned up.